### PR TITLE
fix: reload client on cold start in dev mode

### LINKED ISSUE #2060: Automatically reload client in dev mode on cold start
__What should we do?__

Automatically reload client in dev mode on cold start. Currently, it works only when `client/build/index.html` exists on start.

__Why should we do it?__

We wanna use awesome `@bpmn-io/sr` with Camunda Modeler.

### PR DESCRIPTION
**Pull request by barmac**
The fix is applied via replacement of webpack-dev-server
with electron-reload.

Closes #2060

__Which issue does this PR address?__

Closes #

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by barmac** [comment]
Test this via `npx @bpmn-io/sr "camunda/camunda-modeler#2060-replace-webpack-dev-server-with-electron-reload -c 'npm run dev'

**Comment by andreasgeier** [comment]
Tried on my Mac: works 👍

**Comment by nikku** [comment]
@andreasgeier If you tried, can you approve it?

File: app/dev.js [hunk 1]
```diff
@@ -45,7 +46,12 @@ app.on('app:window-created', async () => {
 });
 
 try {
+
+  // reload on app changes
   require('electron-reloader')(module);
```
Comment (nikku) [inline app/dev.js:51]: What is the benefit of keeping this dude around?

I imagine we can simply reload on any `app/**` changes, too, right.

Just a suggestion, of course.

File: client/package.json [hunk 2]
```diff
@@ -87,11 +87,10 @@
     "style-loader": "^0.23.1",
     "url-loader": "^1.1.1",
     "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.1",
-    "webpack-dev-server": "^3.3.1"
+    "webpack-cli": "^3.3.1"
   },
   "scripts": {
-    "start": "cross-env NODE_ENV=development webpack-dev-server --config ./webpack.config.js --port 3000",
+    "start": "cross-env NODE_ENV=development webpack --watch --config ./webpack.config.js",
```
Comment (nikku) [inline client/package.json:93]: Feels good to get rid of the dev server. :heart:

File: app/dev.js (hunk 1 above)
Comment (barmac) [reply app/dev.js:51]: I think we could also just use `electron-reloader` alone, but that would require moving the dev script upwards. I'll push a change shortly.

Comment (barmac) [reply app/dev.js]: Have a look now. One tool to reload both parts.

**Review by nikku — APPROVED** [review]

**Pull request merged**